src/components/attendee/SearchOverlay/SpaceItem/SpaceItem.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from "react";
+import { useHistory } from "react-router-dom";
+import { Button } from "components/attendee/Button";
+
+import { SPACE_TAXON } from "settings";
+
+import { SpaceWithId } from "types/id";
+
+import { enterSpace } from "utils/url";
+
+import { useWorldById } from "hooks/worlds/useWorldById";
+
+import CN from "../SearchOverlay.module.scss";
+
+export type SpaceItemProps = {
+  space: SpaceWithId;
+  onClick: () => void;
+};
+
+export const SpaceItem: React.FC<SpaceItemProps> = ({ space, onClick }) => {
+  const { world } = useWorldById({ worldId: space.worldId });
+
+  const { push: openUrlUsingRouter } = useHistory();
+
+  const handleEnterSpace = useCallback(() => {
+    enterSpace(world?.slug, space?.slug, {
+      customOpenRelativeUrl: openUrlUsingRouter,
+    });
+    onClick();
+  }, [world?.slug, openUrlUsingRouter, onClick, space?.slug]);
+
+  return (
+    <div>
+      <div className={CN.searchItemResultHeader}>
+        <h3 className={CN.searchItemResultTitle}>
+          {space.name}
+          <span>{SPACE_TAXON.title}</span>
+        </h3>
+      </div>
+      {space.subtitle && (
+        <p className={CN.searchItemResultSubtitle}>{space.subtitle}</p>
+      )}
+      {space.description && (
+        <p className={CN.searchItemResultSubtitle}>{space.description}</p>
+      )}
```

Comment:
are you sure we need to use that description?
have they moved from `space?.config?.landingPageConfig?.description`?

Reply:
replaced with constants from `landingPageConfig` [09146a2](https://github.com/sparkletown/sparkle/pull/3074/commits/09146a2a8a1fd2ccbb351fd1056f0487e6181767)